extension/js/common/platform/store/key-store.ts

```diff
@@ -18,16 +18,19 @@ export class KeyStore extends AbstractStore {
       return keys;
     }
     return keys.filter(ki => {
```

Comment:
Can now be simplified further as `return keys.filter(ki => fingerprints.includes(ki.fingerprint));`